capsule/src/test/java/CapsuleTest.java

```diff
@@ -1040,6 +1040,29 @@ public void testWrapperCapsuleNonCapsuleApp() throws Exception {
         assertEquals("555", getProperty(pb, "p1"));
     }
 
+    @Test
+    public void testCapsuleJvmArgsParsing() throws Exception {
+        Jar jar = newCapsuleJar()
+                .setAttribute("Main-Class", "MyCapsule")
+                .setAttribute("Premain-Class", "MyCapsule")
+                .setAttribute("Application-Class", "com.acme.Foo")
+                .addClass(MyCapsule.class);
+
+        List<String> args = list();
+        List<String> cmdLine = list();
+
+        System.setProperty("capsule.jvm.args", "-Ddouble.quoted.arg=\"escape me\" -Dsingle.quoted.arg='escape me' -Dspace.escaped.arg=escape\\ me");
```

Comment:
I couldn't find a better way than System.setProperty to set capsule.jvm.args. Do you have a suggestion?
In addition, I'm now realizing I didn't reset the property after the test :/ At least I'll do this if you don't have a better solution.